source/pymongo-async-migration.txt

```diff
@@ -0,0 +1,357 @@
+
+          await collection.update_many(...)
+
+   * - ``drop()``
+     - .. code-block:: python
+
+          await collection.drop()
+   
+   * - ``delete_one()``
+     - .. code-block:: python
+
+          await collection.delete_one(...)
+   
+   * - ``delete_many()``
+     - .. code-block:: python
+
+          await collection.delete_many(...)
+   
+   * - ``find_one()``
+     - .. code-block:: python
+          
+          await collection.find_one(...)
+   
+   * - ``estimated_document_count()``
+     - .. code-block:: python
+
+          await collection.estimated_document_count()
+
+   * - ``count_documents()``
+     - .. code-block:: python
+
+          await collection.count_documents(...)
+
+   * - ``create_index()``
+     - .. code-block:: python
+          
+          await collection.create_index(...)
+
+   * - ``create_indexes()``
+     - .. code-block:: python
+          
+          await collection.create_indexes(...)
+
+   * - ``drop_index()``
+     - .. code-block:: python
+
+          await collection.drop_index(...)
+
+   * - ``drop_indexes()``
+     - .. code-block:: python
+
+          await collection.drop_indexes()
+
+   * - ``list_indexes()``
+     - .. code-block:: python
+
+          await collection.list_indexes()
+
+   * - ``index_information()``
+     - .. code-block:: python
+
+          await collection.index_information()
+
+   * - ``list_search_indexes()``
+     - .. code-block:: python
+
+          await collection.list_search_indexes()
+
+   * - ``create_search_index()``
+     - .. code-block:: python
+
+          await collection.create_search_index(...)
+
+   * - ``create_search_indexes()``
+     - .. code-block:: python
+
+          await collection.create_search_indexes(...)
+
+   * - ``drop_search_index()``
+     - .. code-block:: python
+
+          await collection.drop_search_index(...)
+
+   * - ``update_search_index()``
+     - .. code-block:: python
+
+          await collection.update_search_index(...)
+   
+   * - ``options()``
+     - .. code-block:: python
+
+          await collection.options()
+   
+   * - ``aggregate()``
+     - .. code-block:: python
+
+          async for doc in await collection.aggregate(...):
+             ...
+
+   * - ``aggregate_raw_batches()``
+     - .. code-block:: python
+
+          async for batch in await collection.aggregate_raw_batches(...):
+             ...
+
+   * - ``rename()``
+     - .. code-block:: python
+
+          await collection.rename(...)
+
+   * - ``distinct()``
+     - .. code-block:: python
+
+          await collection.distinct(...)
+
+   * - ``find_one_and_delete()``
+     - .. code-block:: python
+
+          await collection.find_one_and_delete(...)
+
+   * - ``find_one_and_replace()``
+     - .. code-block:: python
+
+          await collection.find_one_and_replace(...)
+
+   * - ``find_one_and_update()``
+     - .. code-block:: python
+
+          await collection.find_one_and_update(...)
+
+Migrate From Motor
```

Comment:
I'd like to see the motor -> async pymongo migration as its own dedicated page. The pymongo sync -> async differences clutters the page for motor users.

Reply:
Split the pages into two